### PR TITLE
Design upgrade

### PR DESCRIPTION
**Pull request by subhamX**
## Description
The following PR implements the new frontend design. 

## Screenshots
new_design_dark

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by kgodey** [comment]
I'm excited about this design upgrade. 🎉

**Comment by soccerdroid** [comment]
The file react-frontend/public/data/fdg_input_file.json , could you take instead the one in the django-backend folder?

File: react-frontend/src/components/Graph2D.js [hunk 1]
```diff
@@ -32,6 +49,9 @@ class Graph2D extends React.Component {
         currentZoomLevel: 1, // storing the current zoom level
         // !TODO: to be used for re-setting the state
```
Comment (soccerdroid) [inline react-frontend/src/components/Graph2D.js]: What's the meaning of this comment? I think it's not a ToDo, is it?

File: react-frontend/src/components/Graph2D.js [hunk 2]
```diff
@@ -50,31 +83,30 @@ class Graph2D extends React.Component {
...
                 this.simulateForceGraph(res);
             });
-
     }
 
     render() {
         return (
             <React.Fragment>
-                <SearchFilterBox handleSubmit={this.handleFilterSubmit} />
+                {/* <SearchFilterBox handleSubmit={this.handleFilterSubmit} /> */}
```
Comment (soccerdroid) [inline react-frontend/src/components/Graph2D.js]: please delete this line

**Review by soccerdroid — CHANGES_REQUESTED** [review]

File: react-frontend/src/components/Graph2D.js (hunk 1 above)
Comment (subhamX) [reply react-frontend/src/components/Graph2D.js]: No. It is no longer a todo as we have moved the filtering logic to server-side. Removing it.

**Review by soccerdroid — APPROVED** [review]

**Pull request merged**